### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by mdeff**
Some maintenance on the infrastructure. Most importantly for Travis CI to run again. Should be kept in sync with https://github.com/epfl-lts2/pyunlocbox/pull/27 to facilitate future maintenance by keeping their infra similar.

### CODE REVIEW

**Comment by nperraud** [comment]
@mdeff we should also add python 3.9

File: pygsp/graphs/__init__.py [hunk 1]
```diff
@@ -185,42 +185,35 @@
...
+from .nngraphs.cube import Cube  # noqa: F401
+from .nngraphs.imgpatches import ImgPatches  # noqa: F401
+from .nngraphs.grid2dimgpatches import Grid2dImgPatches  # noqa: F401
+from .nngraphs.sensor import Sensor  # noqa: F401
+from .nngraphs.sphere import Sphere  # noqa: F401
+from .nngraphs.twomoons import TwoMoons  # noqa: F401
+
+
+# Hack: make a list of all graphs
+__all__ = [e for e in dir() if e[0].isupper()]
```
Comment (nperraud) [inline pygsp/graphs/__init__.py:219]: It was not working without this

File: pygsp/graphs/_io.py [hunk 2]
```diff
@@ -373,7 +373,7 @@ def from_graphtool(cls, graph, weight='weight'):
         >>> v = graph.add_vertex()
         >>> eprop = graph.new_edge_property("double")
         >>> eprop[e1] = 0.2
-        >>> eprop[(1, 2)] = 0.9
```
Comment (nperraud) [inline pygsp/graphs/_io.py:376]: this is not working anymore with graph_tool

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
   - 3.6
   - 3.7
-
-matrix:
-  include:
-    - python: 2.7
-      dist: trusty  # "cannot import name QtOpenGL" with xenial
-      cache: pip  # avoid PySide wheel build
+  - 3.8
+  - 3.9
```
Comment (nperraud) [inline .travis.yml:8]: Let us go for 3.9 as well

**Review by nperraud — COMMENTED** [review]
Just a few remark about my modifications.

**Comment by nperraud** [comment]
@mdeff Can I merge it?

File: .travis.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+dist: bionic
```
Comment (mdeff) [inline .travis.yml:1]: Why do we need bionic?

Comment (mdeff) [reply .travis.yml:1]: Because `graph-tool` is not available for xenial anymore. Ok.

File: pygsp/graphs/__init__.py (hunk 1 above)
Comment (mdeff) [reply pygsp/graphs/__init__.py:219]: That's too hacky and brittle. What wasn't working exactly?

File: .travis.yml (hunk 4 above)
Comment (nperraud) [reply .travis.yml:1]: Also an update from time to time is not bad :-)

Comment (mdeff) [reply .travis.yml:1]: Yep. But it requires us to update. Whereas the travis default should auto-update. But they are quite slow to do it...

File: pygsp/graphs/_io.py (hunk 2 above)
Comment (mdeff) [reply pygsp/graphs/_io.py:376]: Good catch thanks!

**Comment by mdeff** [comment]
Finally 🎉

**Comment by nperraud** [comment]
Thanks a lot

**Pull request merged**